solution.cpp

```diff
@@ -1,22 +1,152 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
+ * Copyright &copy; 2015 (Weihan Xiao) <weihan1004@gmail.com>
  * Released under the [MIT License] (http://opensource.org/licenses/MIT)
  * ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
+ * TODO: To transfer numbers into roman numeral type as the output to show user directly.
  */
 
 // TODO: `#include`s and function prototypes
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+using std::cout;
+using std::string;
+using std::iostream;
 
-    return 0;  // success
+void Function()
```

Comment:
functions typically have lower case names -- and also more descriptive names :-)